lib/synapse_pay_rest/models/transaction/transaction.rb

```diff
@@ -152,6 +151,44 @@ def from_response(node, response)
 
       private
 
+      def payload_for_create(node:, to_type:, to_id:, amount:, currency:, ip:, **options)
```

Comment:
Curious about the reasoning to pull this out of the `PayloadCreator` and add this back in here? Both the BatchTransaction and Transaction classes will need to use the same function